src/Raven.Server/Web/Operations/OperationsHandler.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using Raven.Server.Documents;
+using Raven.Server.Routing;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+
+namespace Raven.Server.Web.Operations
+{
+    public class OperationsHandler : DatabaseRequestHandler
+    {
+
+        [RavenAction("/databases/*/operations/next-operation-id", "GET")]
+        public Task GetNextOperationId()
```

Comment:
Why are we using the watch command anyway?

We had to do that in 3.x because we didn't have a way to watch long running operations.
But in 4.0, we don't have a request limit, and we can just have a single request do the work, no?